cylc/flow/scripts/lint.py

```diff
@@ -209,9 +213,17 @@ def check_wallclock_directives(line: str) -> Union[Dict[str, str], bool]:
         >>> this = check_wallclock_directives
         >>> this('    -W 42:22')
         {'directive': '-W 42:22'}
+        >>> this('    -W 42:22/hostname')  # Legit LSF use case
```

Comment:
Looking at https://www.ibm.com/docs/en/spectrum-lsf/10.1.0?topic=o-w-1, I think that this is an LSF wallclock directive which can't be handled by `execution time limit` and therefore is a legitimate case for using `-W` with LSF.

Reply:
Can you do multiple things with this opt?

e.g `-W 00:02 foo=bar`

Reply:
In LSF or PBS?

In LSF, not really.

In PBS, not really - only key=value pairs.

I don't think that the two are mistakeable.

Reply:
Gotcha